components/embl-logo/embl-logo.scss

```diff
@@ -12,20 +12,13 @@
 .embl-logo {
   background-image: url('../assets/embl-logo/assets/logo-medium.svg');
```

Comment:
Design mentioned that this should be just plan `/logo.svg` except for on tablet/mobile, is that worth addressing in this PR?

Reply:
Oh, wait, it looks like that should now work. Disregard!

Reply:
Oh, so the "more dots" one for everything? I thought it was less dots on mobile/tablet -- which this is.

Reply:
It's "more dots on desktop". Quickly scanning your code, it looks like you have it right.

Reply:
yeah - I think the SVGs aren't named well. Perhaps that's a `N.x.x` breaking change to add to the todo list